tests/test_modelgen.py

```diff
@@ -19,9 +55,13 @@ def test_regularization_0size_interval(self):
 
     def test_base_hyper_parameters_reg(self):
         """ Base hyper parameter set should contain regularization. """
-        hyper_parameter_set = modelgen.generate_base_hyper_parameter_set()
+        hyper_parameter_set = modelgen.generate_base_hyper_parameter_set(low_lr=1, 
```

Comment:
Why this change? To me it seems the test is made more complicated (and less readable) without testing anything more or better than it did before. It's not super bad or anything but I don't understand why you changed it. If you wanted to actually see whether the new arguments work than the effect should probably be tested for as well and it that case it is often better to add a test than to replace a simpler one.

Reply:
I agree that is looks less readable, but I tried to get rid of all redundant default settings and hence removed them from the generate_base_hyper_parameter_set() function.